lib/compute/index.js

```diff
@@ -105,6 +105,10 @@ var SCOPES = ['https://www.googleapis.com/auth/compute'];
  * var gce = gcloud.compute();
  */
 function Compute(options) {
+  if (this.config_) {
+    options = util.extendGlobalConfig(this.config_, options);
+  }
+
```

Comment:
I'm thinking this as well as throwing if there is no projectId could go to a single place, so that this just calls "util.normalizeArguments()" or something -- feel free to tell me that's a bad idea.